astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -438,9 +438,7 @@ def test_doubled_quotes(read_csv):
 
     """
     # fmt: off
-    tbl = '\n'.join(['a,b',
-                     '"d""","d""q"',
-                     '"""q",""""'])
+    tbl = '\n'.join(['a,b', '"d""","d""q"', '"""q",""""'])  # noqa: FLY002
```

Comment:
I think we would be much better off using either [`textwrap.dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) or [`inspect.cleandoc()`](https://docs.python.org/3/library/inspect.html#inspect.cleandoc). The code would be even more readable and we wouldn't need the `noqa` instructions. For example:
```python
    tbl = dedent(
        '''\
        a,b
        "d""","d""q"
        """q",""""
        '''
    )
```
Note that it is necessary to use `'''` instead of `"""` here only because the string itself contains `"""`.

Reply:
So I at least can't squash commits on these PRs. Changing this (again) in this PR will just accrue more commits. Do you want to do this in a followup PR? Easy to grep for with the ``# noqa: FLY002``.

Reply:
Or we can ask @pllim to squash?

Reply:
Can you please cherry pick and open new PR from fork? 😬

Reply:
If we want to handle this in a follow-up pull request then I'd prefer if we told Ruff to ignore FLY002 in `.ruff.toml` because that's how we have decided to specify temporarily ignored rules. When I see a `noqa` instruction then I assume that it is meant to be permanent.

Reply:
Closing this PR and opening one with all the back and forth  collapsed to a few elemental commits sounds good. Then we can do it "right". 🤣 a common outcome.

Reply:
> So I at least can't squash commits on these PRs.

And nobody can. We enforce that since the last time when you force pushed on the main repo. `pre-commit` PRs are made on the main repo so they should only be used to update pre-sommit itself, not to refactor the code and squash commits. Please use PRs from your forks for that.

Reply:
The automatic `pre-commit` updates are revealing code that should be refactored, but it looks like it would indeed be better to do that in separate pull requests.